### ISSUE TITLE
HomeAway::API::Errors::InternalServerError 500 Invalid format: "2016-04-01T17:00:00Z" is malformed at "T17:00:00Z"

### ISSUE THREAD

**Issue by hovatterz** [homeaway/homeaway_api_ruby#3]
```
    paginator = @client.search(
      q: '4 bathrooms new york',
      page_size: 30,
      availability_start: '2016-04-01',
      availability_end: '2016-04-08'
    )
```

The gem is adding timezone data before sending the dates which causes an error.

**Comment by cemeyer2** [comment]
Thanks for the report. I will get this fixed tomorrow.

**Comment by cemeyer2** [comment]
fixed in 0fccbf71cdde4b7ae0445605c9f8527d3b5649b5
